### DISCUSSION TITLE
Can I join team?

### DISCUSSION THREAD

**Discussion by khirendra03** [ROM-builders/temporary#10554] [General]
My build: https://cirrus-ci.com/build/5884822953394176

My tree: 
Device specific DT: https://github.com/khirendra03/android_device_asus_X01BD
Common tree: https://github.com/khirendra03/android_device_asus_sdm660-common

**Comment by Apon77** [comment]
> My build: https://cirrus-ci.com/build/5884822953394176
> 
> My tree: Device specific DT: https://github.com/khirendra03/android_device_asus_X01BD Common tree: https://github.com/khirendra03/android_device_asus_sdm660-common

github username give

**Reply by khirendra03** [reply]
khirendra03
